aspnetcore/fundamentals/openapi/using-openapi-documents.md

```diff
@@ -20,7 +20,7 @@ The `Swashbuckle.AspNetCore.SwaggerUi` package provides a bundle of Swagger UI's
 
 * Install the `Swashbuckle.AspNetCore.SwaggerUi` package.
 * Enable the swagger-ui middleware with a reference to the [OpenAPI route registered earlier](xref:fundamentals/openapi/aspnetcore-openapi#customize-the-openapi-endpoint-route).
-* To limit information disclosure and security vulnerability, ***only enable Swagger UI in development environments.***
+* As a security best practice to limit information disclosure, ***OpenAPI user interfaces (Swagger UI, ReDoc, Scalar) should only be enabled in development environments.***
```

Comment:
For Swagger UI, we can be specific here and link to Swagger UI's disclosure about not using the clientSecret config in production. See https://swagger.io/docs/open-source-tools/swagger-ui/usage/oauth2/.

cc: @GrabYourPitchforks @blowdart for thoughts on the verbiage here

Reply:
> 
> cc: @GrabYourPitchforks @blowdart for thoughts on the verbiage here